src/components/20-molecules/top-content-bar/index.js

```diff
@@ -57,6 +57,8 @@ class AXATopContentBar extends InlineStyles {
       __VERSION_INFO__,
       this
     );
+
+    this.style.display = this.closed ? 'none' : 'block';
```

Comment:
This unfortunately is an anti-pattern here. Styles and classes on the host element are supposed to be managed by the application only, this would introduce potential conflicts.

The proper way to solve FOUC is to work with what the (browser) platform provides: see our https://github.com/axa-ch-webhub-cloud/pattern-library/blob/develop/ARCHITECTURE.md section "13. FOUC" for How.

Reply:
This is awesome! I just added 
```
<style type="text/css">
    axa-top-content-bar:not(:defined) {
        display: none;
    }
</style>
```
above the axa-top-content-bar element and it works.